src/cct.c

```diff
@@ -206,8 +206,8 @@ int main(int argc, char **argv) {
     PJ_COORD point;
     PJ_PROJ_INFO info;
     OPTARGS *o;
-    char *buf;
-    int nfields = 4, direction = 1, skip_lines = 0, verbose;
```

Comment:
I think you can simplify the control flow somewhat by doing something like

    char blank_comment[] = {""};
    char *comment = blank_comment;

here